### PR TITLE
Add MountSensitive in CSIproxy mounter

### PR DESCRIPTION
**Pull request by jingxu97**
Add MountSensitive

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jingxu97** [comment]
cc @saad-ali @msau42 @verult

**Comment by msau42** [comment]
Thanks for fixing this. Could you also add the windows build into the unit or verify job so that it is a blocking test?

**Comment by jingxu97** [comment]
> Thanks for fixing this. Could you also add the windows build into the unit or verify job so that it is a blocking test?

I added it to verify-all.sh. PTAL. THanks!

File: hack/verify-all.sh [hunk 1]
```diff
@@ -20,5 +20,7 @@ set -o pipefail
 
 PKG_ROOT=$(git rev-parse --show-toplevel)
 
-${PKG_ROOT}/hack/verify-gofmt.sh
-${PKG_ROOT}/hack/verify-govet.sh
+#${PKG_ROOT}/hack/verify-gofmt.sh
```
Comment (msau42) [inline hack/verify-all.sh:23]: why are these commented out?

Comment (jingxu97) [reply hack/verify-all.sh:23]: sorry, my mistake

**Pull request merged**